packages/service-library/src/servicelib/rabbitmq/_models.py

```diff
@@ -11,6 +11,11 @@
 
 MessageHandler = Callable[[Any], Awaitable[bool]]
 
+ExchangeName: TypeAlias = str
```

Comment:
MINOR: should we add some constraints to these names? i.e. all identifiers should have minimum number of letters and allowed characters. That is the purpose of all constraint identifiers like `IDStr` etc